.github/workflows/trufflehog.yml

```diff
@@ -0,0 +1,16 @@
+name: trufflehog
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  run-trufflehog:
```

Comment:
Maybe just trufflehog, to be consistent with the format used for the gitleaks job?

Reply:
```suggestion
  trufflehog:
```